eox_nelp/course_api/serializers.py

```diff
@@ -163,85 +38,79 @@ def get_overview(self, course_overview):
         # Note: This makes a call to the modulestore, unlike the other
         # fields from CourseSerializer, which get their data
         # from the CourseOverview object in SQL.
-        self.raw_overview = CourseDetails.fetch_about_attribute(course_overview.id, 'overview')
+        self.raw_overview = CourseDetails.fetch_about_attribute(course_overview.id, "overview")
         return self.raw_overview
+
     overview_object = serializers.SerializerMethodField()
+
     def get_overview_object(self, course_overview):
         """Get an object that parse the html information and extract
         `about_description`,`staff`, `prereqs` and `faqs` of course_overview.
          https://edx.readthedocs.io/projects/edx-open-learning-xml/en/latest/about/overview.html
         """
 
         html_str = self.raw_overview
+
         def decompose_str_p(p_tag):
             """Join all str in a paragraph tag in one string."""
             generator = p_tag.strings
             string_list = [string for string in generator]
-            return ''.join(string_list)
+            return "".join(string_list)
 
         def get_titles_and_paragraphs(tag, title_tag_atr, p_tag_atr, title_name="titles", p_name="paragraphs"):
             """Get an object represetantion after searching by desired titles and paragraphs tags"""
             try:
                 return {
-                title_name:  [title.string for title in tag.find_all(title_tag_atr) ],
-                p_name: [decompose_str_p(p_tag) for p_tag in tag.find_all(p_tag_atr) ]
+                    title_name: [title.string for title in tag.find_all(title_tag_atr)],
+                    p_name: [decompose_str_p(p_tag) for p_tag in tag.find_all(p_tag_atr)],
                 }
 
             except:
-                return {
-                    title_name: [],
-                    p_name: [],
-                }
+                return {title_name: [], p_name: []}
 
         def get_source_images(tag, image_name="image_url"):
+            """Get src images from tag"""
             try:
                 return {
-                image_name: [self.context['request'].build_absolute_uri(title.get('src')) for title in tag.find_all('img') ]
+                    image_name: [
+                        self.context["request"].build_absolute_uri(title.get("src")) for title in tag.find_all("img")
+                    ]
```

Comment:
what are the possible exceptions ?

Reply:
[e9cb44b](https://github.com/eduNEXT/eox-nelp/pull/21/commits/e9cb44be90207c26a2342d359822b4512ee086d2)

Reply:
not need the try-except block code with the if. ![2023-02-20_12-14](https://user-images.githubusercontent.com/51926076/220392697-03842612-0453-4c94-8289-4ce3089ec93e.png)